src/Resolver.js

```diff
@@ -354,13 +354,24 @@ class Resolver {
 
       alias = this.lookupAlias(aliases, filename);
     } else {
-      // It is a node_module. First try the entire filename as a key.
-      alias = aliases[filename];
-      if (alias == null) {
-        // If it didn't match, try only the module name.
-        let parts = this.getModuleParts(filename);
-        alias = aliases[parts[0]];
-        if (typeof alias === 'string') {
+      let parts = this.getModuleParts(filename);
+
+      alias = aliases[parts[0]];
+
+      // If the alias starts with . or ./ it doesn't point
+      // to a node module instead to a folder or file of the project
+      // convention of webpack https://webpack.js.org/configuration/resolve/
+      if (typeof alias === 'string') {
+        const isAliasToFolder = alias.startsWith('.') || alias.startsWith('/');
+
+        if (isAliasToFolder) {
+          alias = alias + '/' + path.join(...parts.slice(1));
+
+          if (alias.startsWith('/')) {
+            alias = '.' + alias;
```

Comment:
Shouldn't this resolve relative to the project root?

Reply:
This sounds like a good idea to me. I have to confess, I just learned the difference about distinction of root directory / project directory in parcel. Thank you very much 👍 :)